include/onnxruntime/core/framework/execution_provider.h

```diff
@@ -182,6 +182,12 @@ class IExecutionProvider {
   */
   virtual common::Status Sync() const { return Status::OK(); }
 
+  /**
+     Set graph annotation for saving/retriving executable graphs (e.g., cuda graph).
+     Currently only CUDA execution provider supports it.
```

Comment:
Nit: Not related to your change, but can you update this line "Currently only CUDA execution provider..." in the other Graph related methods in this file? Other EPs have supported Graphs but comments have not been updated in this file to reflect that.

Reply:
sure